### DISCUSSION TITLE
[Feature Request] GRID, Container option in Layout

### DISCUSSION THREAD

**Discussion by narottam-dev** [heroui-inc/heroui#4728] [Feature requests]
### Is your feature request related to a problem? Please describe.

As per the UI library Next UI has everything what a application need but as there is no option for Grid in Layout we have to choose library like Ant D, Material UI over this. If Next UI also can provide Grid option in Layout it will be a finalize library for us to choose this over any other library.

### Describe the solution you'd like

As a user I expect Row, Column, Responsive for whole layout or component features.

### Describe alternatives you've considered

NA

**Comment by linear** [comment]
<p><a href="https://linear.app/nextui-inc/issue/ENG-1463/[feature-request]-grid-in-layout">ENG-1463 [Feature Request] GRID in Layout</a></p>

**Comment by successbyte** [comment]
Great idea 
we use Quasar (Vue) in our company because it has all components and some Grid and Flex layouts
